pkg/eventer/event_recorder.go

```diff
@@ -8,34 +8,40 @@ import (
-	EventReasonFailedToPause        string = "Failed"
-	EventReasonFailedToDelete       string = "Failed"
-	EventReasonFailedToWipeOut      string = "Failed"
-	EventReasonFailedToGet          string = "Failed"
-	EventReasonFailedToInitialize   string = "Failed"
-	EventReasonFailedToList         string = "Failed"
-	EventReasonFailedToResume       string = "Failed"
-	EventReasonFailedToSchedule     string = "Failed"
-	EventReasonFailedToStart        string = "Failed"
-	EventReasonFailedToUpdate       string = "Failed"
-	EventReasonIgnoredSnapshot      string = "IgnoredSnapshot"
-	EventReasonInitializing         string = "Initializing"
-	EventReasonInvalid              string = "Invalid"
-	EventReasonInvalidUpdate        string = "InvalidUpdate"
-	EventReasonResuming             string = "Resuming"
-	EventReasonSnapshotFailed       string = "SnapshotFailed"
-	EventReasonStarting             string = "Starting"
-	EventReasonSuccessfulCreate     string = "SuccessfulCreate"
-	EventReasonSuccessfulPause      string = "SuccessfulPause"
-	EventReasonSuccessfulResume     string = "SuccessfulResume"
-	EventReasonSuccessfulWipeOut    string = "SuccessfulWipeOut"
-	EventReasonSuccessfulSnapshot   string = "SuccessfulSnapshot"
-	EventReasonSuccessfulValidate   string = "SuccessfulValidate"
-	EventReasonSuccessfulInitialize string = "SuccessfulInitialize"
+	EventReasonCreating                string = "Creating"
+	EventReasonPausing                 string = "Pausing"
+	EventReasonWipingOut               string = "WipingOut"
+	EventReasonFailedToCreate          string = "Failed"
+	EventReasonFailedToPause           string = "Failed"
+	EventReasonFailedToDelete          string = "Failed"
+	EventReasonFailedToWipeOut         string = "Failed"
+	EventReasonFailedToGet             string = "Failed"
+	EventReasonFailedToInitialize      string = "Failed"
+	EventReasonFailedToList            string = "Failed"
+	EventReasonFailedToResume          string = "Failed"
+	EventReasonFailedToSchedule        string = "Failed"
+	EventReasonFailedToStart           string = "Failed"
+	EventReasonFailedToUpdate          string = "Failed"
+	EventReasonIgnoredSnapshot         string = "IgnoredSnapshot"
+	EventReasonInitializing            string = "Initializing"
+	EventReasonInvalid                 string = "Invalid"
+	EventReasonInvalidUpdate           string = "InvalidUpdate"
+	EventReasonResuming                string = "Resuming"
+	EventReasonSnapshotFailed          string = "SnapshotFailed"
+	EventReasonStarting                string = "Starting"
+	EventReasonSuccessfulCreate        string = "SuccessfulCreate"
+	EventReasonSuccessfulPause         string = "SuccessfulPause"
+	EventReasonSuccessfulMonitorAdd    string = "SuccessfulMonitorAdd"
+	EventReasonSuccessfulMonitorDelete string = "SuccessfulMonitorDelete"
+	EventReasonSuccessfulMonitorUpdate string = "SuccessfulMonitorUpdate"
+	EventReasonFailedToAddMonitor      string = "MonitorAddFailed"
```

Comment:
Also, I suggest to rename "SuccessfulMonitorAdd" to "MonitorAdded". What do you think? @TamalSaha